srcpkgs/devilutionX/template

```diff
@@ -1,16 +1,25 @@
 # Template file for 'devilutionX'
 pkgname=devilutionX
-version=1.2.1
+version=1.4.1
 revision=1
 build_style=cmake
-configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
-makedepends="SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel"
+configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DBUILD_TESTING=OFF -DDISABLE_ZERO_TIER=ON"
```

Comment:
```suggestion
configure_args="-DVERSION_NUM=$version -DBUILD_TESTING=OFF -DDISABLE_ZERO_TIER=ON"
```

`BINARY_RELEASE` is removed upstream.

Reply:
This option is no longer needed. There is a lot of inconstancy in the source code of devilutionX, a lot of code is removed and added, it makes compiling difficult.